### PR TITLE
DCKUBE-580 Add some content for DEVELOPMENT.md document

### PR DESCRIPTION
**Pull request by nghazali**

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -14,6 +47,45 @@ Repository contains repository hook definition to auto-generate chart documentat
 When you have the pre-commit hook installed and `commit` a file, it will run the pre-commit hook. If there is a change generated in the `README.md`, the hook will fail and notify you about the new change. You can review the changes and run the `commit` again, now it should pass.
 
 
+## Build and test the project
```
Comment (errcode1202) [inline DEVELOPMENT.md:50]: Im really not sure if we should be advertising this stuff. `helm_install.sh` is something **we use only for CI**, and isnt used for "development" of the charts per se. It's caused problems/confusion in the past and Ive stopped using it myself. To be honest if a turn key solution is what we're trying to provide here then we should wait for the official Terraform scripts.

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+## Prerequisites 
+Before deploy and setup Atlassian's Data Center products the following resources are required:
+* A Kubernetes cluster
+* An Ingress Controller
+* A database
+* Have a `shared-home` volume
+* Setup `local-home` volume
+
+To see how to create and setup those resources follow [Prerequisites Documentation](https://atlassian-labs.github.io/data-center-helm-charts/userguide/PREREQUISITES/).
+
```
Comment (errcode1202) [inline DEVELOPMENT.md:35]: A lot of this content feels like its duplicated from the PREREQUISITE doc Would it not be better to add this detail to it and then reference it from here?

**Review by errcode1202 — CHANGES_REQUESTED** [review]

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -14,6 +47,45 @@ Repository contains repository hook definition to auto-generate chart documentat
...
+```shell script
+./src/test/scripts/helm_install.sh target/config/[product]/helm_parameters
+```
+
+To uninstall the test product and cleanup the resources run `helm_uninstall.sh`:
+
+```shell script
+./src/test/scripts/helm_uninstall.sh target/config/[product]/helm_parameters
+```
+
```
Comment (errcode1202) [inline DEVELOPMENT.md:88]: Im really not sure if we should be advertising this stuff. helm_install.sh is something we use only for CI, and isnt used for "development" of the charts per se. It's caused problems/confusion in the past and Ive stopped using it myself. To be honest if a turn key solution is what we're trying to provide here then we should wait for the official Terraform scripts.

**Review by errcode1202 — CHANGES_REQUESTED** [review]

**Comment by nghazali** [comment]
As result of the team discussion I am closing this PR.

**Pull request closed without merging**